contracts/governance/Vesting/fouryear/FourYearVesting.sol

```diff
@@ -55,10 +56,26 @@ contract FourYearVesting is FourYearVestingStorage, Proxy {
 	}
 
```

Comment:
L42-43 above
```
require(_cliff == 4 weeks, "invalid cliff");
require(_duration == 156 weeks, "invalid duration");
```
these are constants semantically - no  need to consume storage space
and addresses verification - pls use OZ Address lib to check if the address is a contract or verify required interface by using EIP-165

Reply:
Done